deviceComunicationDocumentation.MD

```diff
@@ -4,116 +4,107 @@ Tento dokument slouží k popisu chování jednotlivých zařízení a k popisu
-následujícím způsobem:
+Kapacitory v zařízeních se liší podle používané sítě zařízení. Pro hard restart je potřeba postupovat následujícím způsobem:
 
-* Pro Lora zařízení stačí vyjmout baterii a vložit ji zpět
-* Pro NBIot zařízení je potřeba vyjmout baterii stisknout restart tlačítko a poté znovu vložit baterii
+* Pro LoRa zařízení stačí vyjmout baterii a vložit ji zpět
+* Pro NB-IoT zařízení je potřeba vyjmout baterii stisknout restart tlačítko a poté znovu vložit baterii
 
-Poté co je zařízení restartováno provádí následující kroky (pokud nedojde k erroru (TODO odkaz)):
+Poté co je zařízení restartováno provádí následující kroky (pokud nedojde k chybě):
 
-1. Dioda blikne 1x aby oznámila restart
-2. Dioda blikne 1x aby oznámila konec inicializace
-3. Dioda blikne 10x aby oznámila přechod do transportního režimu
-4. Odeslání transport zprávy (TODO odkaz)
-5. Přechod do transportního režimu
+1. Dioda blikne 1x pro oznámení restartu
+2. Dioda blikne 1x pro oznámení ukončení inicializace
+3. Dioda blikne 10x pro oznámení přechodu do transportního režimu
+4. Odeslání [transport zprávy](#transport)
+5. Přechod do [transportního režimu](#transportní-režim)
 
-Při hard restartu se na defaultní hodnoty nastaví všechny informace které zařízení udržuje v paměti.
+Při hard restartu se na defaultní hodnoty nastaví všechny informace, které zařízení udržuje v paměti.
 
-Pokud je do zařízení vložena baterie která není plně nabitá (2,5V - 2,9V) tak zařízení detekuje error a chová se podle
-popisu v sekci
-XXX (TODO odkaz) a zařízení se nedostane do transportního režimu.
+Pokud je do zařízení vložena baterie, která není plně nabitá (2,5 V - 2,9 V), zařízení detekuje error a chová se podle
+popisu v sekci [Fatální chyba](#fatální-chyba). Zařízení nepřechází do [transportního režimu](#transportní-režim).
 
-Pokud zařízení detekuje jakýkoliv jiný error tak zařízení také postupuje podle informací popsaných v sekci o errorech (
-TODO odkaz).
+Pokud zařízení detekuje jakýkoliv jiný error, tak zařízení postupuje podle informací popsaných v sekci o [chybových zprávách].
 
 ### Kontrola stavu baterie
 
-Zařízení při restartu (přesněji při inicializaci) kontroluje stav baterie. Pokud je stav baterie nízký (<2,5) tak
-se na místo restartu přepne do transportního režimu.
+Zařízení při restartu, resp. při inicializaci kontroluje stav baterie. Pokud je napětí baterie nižší než 2,5 V, zařízení se přepne do transportního režimu.
 
 Stav baterie se také kontroluje pokaždé když zařízení odesílá zprávu nebo měří data. Pokud je v této situaci
-naměřen nízký stav baterie tak se zařízení restartuje. Při inicializaci je pak zjištěn nízký stav baterie
-a zařízení se přepne do transportního režimu.
+naměřen nízký stav baterie, tak se zařízení přepne do transportního režimu.
+
+### Transportní režim
+
+Transportní režim je speciální stav zařízení, ve kterém zařízení nekomunikuje a šetří baterii. Obvykle se používá pro přepravu nebo skladování zařízení.
+
+### Disconnect režim
+
+TODO
```

Comment:
Doplnit popis disconnect režimu - nebo odkázat na popis fatální chyby? (tam to vypadá že je něco takového popsané)

Reply:
disconnect rezim uplne odstranen